src/css-grids/cssgrids-styles.css

```diff
@@ -0,0 +1,296 @@
+    width: 100vw;
+    display: grid;
+    grid-template-columns: 20% repeat(4, 1fr);
+    grid-template-rows: 8% 14% 7% 25% 1fr ;
+    grid-row-gap: 15px;
+    grid-column-gap: 20px;;
+}
+
+.topnavbar {
+    background-color: black;
+    grid-row: 1/2;
+    grid-column: 1/-1;
+    opacity: 65%;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    color: white;
+}
+
+.topnavbar i {
+    padding-right: 25px;
+    padding-left: 25px;
+}
+
+.header1 {
+    background-color: white;
+    grid-column: 2/-1;
+    grid-row: 1/3;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-end;
+    border-radius: 6px;
+}
+
+.header1 h1 {
+    padding-left: 10px;
+    text-decoration: none;
+    font-size: 36px;
+    color: rgb(83, 83, 83);
+    padding-left: 20px;
+    font-weight: 600;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.header1 h4 {
+    padding-left: 10px;
+    text-decoration: none;
+    font-size: 20px;
+    color: rgba(83, 83, 83, 0.822);
+    padding-left: 20px;
+    font-weight: 600;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    padding-bottom: 2px;
+}
+
+
+.directory-bar {
+    background-color: white;
+    opacity: 80%;
+    grid-column: 2/6;
+    grid-row: 3/4;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    font-weight: 500;
+    padding: 10px;
+    border-radius: 6px;
+}
+
+.directory-links {
+    text-decoration: none;
+}
+
+.aside-toolbar {
+    background-color: #21bdca;
+    opacity: 85%;
+    grid-column: 1/2;
+    grid-row: 1/-1;
+    display: grid;
+    grid-template-rows: 16% repeat(8, 8%) 1fr;
+    grid-template-columns: 100%;
+}
+
+.toolbar-buttons {
+    text-decoration: none;
+    color: white;
+    display: flex;
+    padding-left: 20px;
+    align-items: center;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    border-bottom: 1px solid #167e88;
+    border-top: 1px solid #78e6f0;
+}
+
+.toolbar-buttons:not(p):hover {
+    background-color: #17a0a8;
+    border-top: #167e88;
+}
+
+p.toolbar-buttons {
+    border: none;
+}
+
+.main-dash {
+    background-color: white;
+    opacity: 87%;
+    grid-column: 2/4;
+    grid-row: 4/-1;
+    border-radius: 6px;
+}
+
+.side-widgets {
+    background-color: white;
+    opacity: 85%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 6px;
+}
+
+/*To do list*/
+#checklist {
+}
+
+#checklist ul {
+    border: 2px solid darkgray;
+    background-color: rgba(211, 211, 211, 0.384);
+    padding: 0;
+    list-style-type: none;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: ;
+    height: 80%;
+    width: 80%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    flex-direction: column;
+}
+
+#checklist ul li {
+    margin: 5px;
+}
+
+/*Clock widget*/
```

Comment:
yes to these notes that tell you where a new component starts! very helpful! 
don't be afraid to make it even more noticable. `/* !!!! CLOCK WIDGET !!!! */` . 
whatever works for you!